packages/components/src/mobile/notification-sheet/index.native.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { Platform, View, Text } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { BottomSheet, Icon } from '@wordpress/components';
+import { withPreferredColorScheme } from '@wordpress/compose';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+function NotificationSheet( { title, getStylesFromColorScheme, isVisible, onClose, type = 'singular', withMessage = true, withQuotes = true } ) {
+	const infoTextStyle = getStylesFromColorScheme( styles.infoText, styles.infoTextDark );
+	const infoTitleStyle = getStylesFromColorScheme( styles.infoTitle, styles.infoTitleDark );
+	const infoDescriptionStyle = getStylesFromColorScheme( styles.infoDescription, styles.infoDescriptionDark );
+	const infoSheetIconStyle = getStylesFromColorScheme( styles.infoSheetIcon, styles.infoSheetIconDark );
+
+	// translators: %s: Name of the block
+	const titleFormatSingular = Platform.OS === 'android' ? __( '%s isn\'t yet supported on WordPress for Android' ) :
+		__( '%s isn\'t yet supported on WordPress for iOS' );
+
+	const titleFormatPlural = Platform.OS === 'android' ? __( '%s aren\'t yet supported on WordPress for Android' ) :
+		__( '%s aren\'t yet supported on WordPress for iOS' );
+
+	const titleFormat = type === 'plural' ? titleFormatPlural : titleFormatSingular;
+
+	const titleType = withQuotes ? `'${ title }'` : title;
+
+	const infoTitle = sprintf(
+		titleFormat,
+		titleType,
+	);
+
+	return (
+		<BottomSheet
+			isVisible={ isVisible }
+			hideHeader
+			onClose={ onClose }
+		>
+			<View style={ styles.infoContainer } >
+				<Icon icon="editor-help" color={ infoSheetIconStyle.color } size={ styles.infoSheetIcon.size } />
+				<Text style={ [ infoTextStyle, infoTitleStyle ] }>
+					{ infoTitle }
+				</Text>
+				{ withMessage && <Text style={ [ infoTextStyle, infoDescriptionStyle ] }>
```

Comment:
`withMessage` doesn't clearly convey which message it's talking about. This is more about an `unsupported block` vs an `unsupported feature`. The prop name should better reflect that, although I'm thinking I might even have two different components:

- This one, being `UnsupportedFeatureNotice`, replacing the message with `{ children }`
- A `UnsupportedBlockNotice` that would use the previous one and add the message as a child.

Reply:
Outdated - `NotifcationSheet` is removed since we are not using bottom sheet to inform user about missing theme colors.